src/components/overworld/platforms/MainPlatform.tsx

```diff
@@ -20,6 +20,13 @@ export default function MainPlatform({
 	buttonReference,
 	isPlatformFixed,
 }: MainPlatformOptions): JSX.Element {
+	let shipment_infotafel_pos_y: number = isPlatformFixed?.shipment ? 0 : -5;
+	let design_infotafel_pos_y: number = isPlatformFixed?.design ? 0 : -5;
```

Comment:
```suggestion
	let design_infotafel_pos_y: number = isPlatformFixed?.design ? 0 : -10;
```

Because you can see the object still

Reply:
For every platform